addons/crm/models/crm_team.py

```diff
@@ -605,10 +605,7 @@ def action_opportunity_forecast(self):
     @api.model
     def _action_update_to_pipeline(self, action):
         user_team_id = self.env.user.sale_team_id.id
-        if user_team_id:
-            # To ensure that the team is readable in multi company
-            user_team_id = self.search([('id', '=', user_team_id)], limit=1).id
```

Comment:
Why removing that part ?

Reply:
Ah ok, no fine for me, less code is better :) just update commit message then, thanks :)